### PR TITLE
Borg fire extinguishers and welding tools now get refilled

### LINKED ISSUE #9901: Cyborg extinguisher
**Problem Description**:
So I was a medical borg and someone (gee I wonder who it was) released a virus that caused people to combust every now and then. I put people out for a while until I ran out and discovered that there is no way to refill the extinguisher.

**What did you expect to happen**: 
Most things refill at the cyborg recharge station, such as all the medical supplies.

The extinguisher seems to be the exception.

**Why is this bad/What are the consequences:**
I kinda just had to twiddle my borgy thumbs while people burned to death.

**Steps to reproduce the problem**:
I wasn't doing anything out of the ordinary, again, just running around medbay trying to put people out.

### PR DESCRIPTION
**Pull request by farie82**
**What does this PR do:**
Removes the inconsistency with cyborg recharging.
It now also recharges fire extinguishers and welding tools. Both easy to refill but it was a bit odd.

fixes #9901

**Changelog:**
:cl: Farie82
tweak: Cyborgs can now refill welding tools and fire extinguishers using the recharging station
/:cl:

### CODE REVIEW

**Comment by uzername** [comment]
And what about that light replacer? It was not refilling for janitor cyborg.

File: code/game/machinery/rechargestation.dm [hunk 1]
```diff
@@ -220,6 +220,16 @@
 						var/i = 1
 						for(1, i <= coeff, i++)
 							LR.Charge(occupant)
+					//Fire extinguisher
+					if(istype(O, /obj/item/extinguisher))
+						var/obj/item/extinguisher/ext = O
+						if(ext.reagents.get_reagent_amount("water") < ext.max_water)
+							ext.reagents.add_reagent("water", 5 * coeff)
+					//Welding tools
+					if(istype(O,/obj/item/weldingtool))
```
Comment (Birdtalon) [inline code/game/machinery/rechargestation.dm]: Spacing.

File: code/game/machinery/rechargestation.dm [hunk 2]
```diff
@@ -220,6 +220,16 @@
 						var/i = 1
 						for(1, i <= coeff, i++)
 							LR.Charge(occupant)
+					//Fire extinguisher
+					if(istype(O, /obj/item/extinguisher))
+						var/obj/item/extinguisher/ext = O
+						if(ext.reagents.get_reagent_amount("water") < ext.max_water)
```
Comment (Birdtalon) [inline code/game/machinery/rechargestation.dm]: You could use `check_and_add()` here.

**Comment by farie82** [comment]
@uzername they do recharge. Just tested it. They are slow to recharge though

File: code/game/machinery/rechargestation.dm (hunk 2 above)
Comment (farie82) [reply code/game/machinery/rechargestation.dm]: Nice I've used it now. Thanks!

**Comment by variableundefined** [comment]
Someday down the line it might be better off to port over whatever tg uses to charge the cyborg directly.

**Pull request merged**